config/rubocop/.rubocop.yml

```diff
@@ -512,7 +512,7 @@ Style/CommandLiteral:
 Style/CommentAnnotation:
   Description: >-
                  Checks formatting of special comments
-                 (TODO, FIXME, OPTIMIZE, HACK, REVIEW).
+                 (T-ODO, F-IXME, O-PTIMIZE, H-ACK, R-EVIEW).
```

Comment:
What do you think about replacing the description with:

> Checks formatting of annotation comments.

This is the language the Ruby style guide users and sufficiently explains what the check is. I think a hyphen in the word would be confusing to a reader.

See: https://github.com/bbatsov/ruby-style-guide#comment-annotations